op-chain-ops/genesis/genesis.go

```diff
@@ -52,8 +52,8 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		TerminalTotalDifficulty:       big.NewInt(0),
 		TerminalTotalDifficultyPassed: true,
 		Optimism: &params.OptimismConfig{
+			L1FeeRecipient:     config.OptimismL1FeeRecipient,
```

Comment:
Maybe we should also be updating the config name here - the `L1FeeRecipient` should receive all fees collected on L2 except for the basefee (which goes to the `BaseFeeRecipient`